_inc/client/components/connect-button/style.scss

```diff
@@ -0,0 +1,17 @@
+@import "../../scss/variables/_colors";
+
+// ==========================================================================
+// Buttons
+// ==========================================================================
+
+.jp-jetpack-connect__button {
+	background: $green-primary;
+    border-color: $green-secondary;
+    color: $white;
```

Comment:
Indentation seems to be using different styles here. Let's use tabs please.

Reply:
That is my mistake. Thanks for catching!